.github/workflows/build.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         with:
           version: v0.68.0
           static: ${{ matrix.os.static }}
-      - run: npm install -g markdownlint-cli
+      - run: npm install -g markdownlint-cli dprint
```

Comment:
💡 distribution via npmjs.com, that's neat.

Reply:
Bit of a shame for a rust tool... but it's work well in other places.